src/xml2js.coffee

```diff
@@ -92,6 +106,7 @@ class exports.Builder
   buildObject: (rootObj) ->
     attrkey = @options.attrkey
     charkey = @options.charkey
+    cdata = @options.cdata
```

Comment:
We have to do it like this as the `this` context changes. See this mini-example:

```javascript
function foo(){};
foo.prototype.doit = function() {
  this.cdata = 'cdata';
  var bar = function() {
    console.log('this.cdata:', this.cdata);
  }
  bar();
};
var zip = new foo();
zip.doit();
```